pkg/kubelet/lifecycle/predicate.go

```diff
@@ -150,6 +153,27 @@ func (w *predicateAdmitHandler) Admit(attrs *PodAdmitAttributes) PodAdmitResult
 	}
 }
 
+func (w *predicateAdmitHandler) generalPredicates(pod *v1.Pod,
+	nodeInfo *schedulernodeinfo.NodeInfo) (bool, schedulerframework.PluginToStatus, error) {
+	state := schedulerframework.NewCycleState()
+	preFilterStatus := w.schedulerFramework.RunPreFilterPlugins(context.TODO(), state, pod)
+	if !preFilterStatus.IsSuccess() {
+		var err error
+		if !preFilterStatus.IsUnschedulable() {
+			err = preFilterStatus.AsError()
+		}
+		return false, schedulerframework.PluginToStatus{"prefilter": preFilterStatus}, err
```

Comment:
Are we going to use the `name` somewhere? If not, we can simply this data structure (map) by simply using the status.